Dockerfile.release

```diff
@@ -0,0 +1,37 @@
+# This dockerfile is distinct from the kb-sdk generated one. This builds an image to be deployed
+# as a core service, not a KBase app 
+FROM kbase/kb_jre
+
+# These ARGs values are passed in via the docker build command
+ARG BUILD_DATE
+ARG VCS_REF
+ARG BRANCH=develop
+
+RUN mkdir /kb/module
+
+COPY deployment/ /kb/deployment/
+COPY resources /kb/module/resources/
+
+RUN cd /kb/deployment/lib && unzip /kb/deployment/jettybase/webapps/root.war
```

Comment:
The deployment container is stripped down and doesn't include a lot of things ( the jar command included). Make and ant are not in the deployment container. In other repos (such as Workspace) the war file has been created in the main build environment as well, so I'm just going along with existing practice.

Reply:
So why the unzip?

Reply:
Unzip expands the war file so that the search_tools.sh script can include them in the search path. There doesn't seem to be a way to include a WAR file as part of the classpath to java, and the KBaseSearchEngine.jar file doesn't seem to include dependencies that are in the jars repo.

Reply:
> KBaseSearchEngine.jar file doesn't seem to include dependencies that are in the jars repo
Well, you wouldn't expect it to, generally speaking. It's much more common to pack everything into a war file, although there are jars packed with dependencies.

The solution to the problem is using the ant target to build the search_tools script which sets up the classpath appropriately. Since you don't have ant, I guess this hack is the way to do it...